packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { MouseEventHandler } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: string;
+  rightIcon?: string;
+  buttonStyle?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+/**
+ * Cyberstorm Button component
+ */
+export const Button: React.FC<ButtonProps> = (props) => {
+  const { label, leftIcon, rightIcon, buttonStyle, onClick } = props;
+  const stylesImport = styles;
+
+  return (
+    <button className={"button button__" + buttonStyle} onClick={onClick}>
```

Comment:
This should define `type="button"`. Or allow passing it as a prop, but defaulting to `"button"` if no prop is passed.

Reply:
This is a good catch, as button should always have a `type` defined. By default it should be `button` if no prop is passed, but other possible values are `submit` and `reset` which handle `form` operations, and should be possible to pass as a prop.